scripts/packages/packager/Dockerfile

```
@@ -11,7 +11,7 @@ RUN apt-get update && \
 
 # compile, install pkg tool for linux
 RUN	mkdir $PKG_DIR; cd $PKG_DIR; \
-	go install github.com/goreleaser/nfpm/v2/cmd/nfpm@v2.32.0; \
+	go install github.com/goreleaser/nfpm/v2/cmd/nfpm@vv2.35.3; \
```

Is the extra `v` a mistake?
```suggestion
	go install github.com/goreleaser/nfpm/v2/cmd/nfpm@v2.35.3; \
```

---

yup, thanks